parts/kuberneteswindowssetup.ps1

```diff
@@ -210,7 +210,7 @@ try
     # if the podCIDR has not yet been assigned to this node, start the kubelet process to get the podCIDR, and then promptly kill it.
     if (-not `$podCidrDiscovered)
     {
-        `$argList = @("--hostname-override=`$global:AzureHostname","--pod-infra-container-image=kubletwin/pause","--resolv-conf=""""","--api-servers=https://`${global:MasterIP}:443","--kubeconfig=c:\k\config")
+        `$argList = @("--hostname-override=`$global:AzureHostname","--pod-infra-container-image=kubletwin/pause","--resolv-conf=""""","--api-servers=https://`${global:MasterIP}:443","--kubeconfig=c:\k\config","--enable-cri=false")

```

Comment:
This should be fine but there is a bit duplication. Could you try the following or something like:
```
$global:KubeletArgListBase = @("--hostname-override=`$global:AzureHostname","--pod-infra-container-image=kubletwin/pause","--resolv-conf=""""""""","--api-servers=https://`${global:MasterIP}:443","--kubeconfig=c:\k\config")
$global:KubeletCommandLineBase = @"
c:\k\kubelet.exe --hostname-override=`$global:AzureHostname --pod-infra-container-image=kubletwin/pause --resolv-conf="" --allow-privileged=true --enable-debugging-handlers --api-servers=https://`${global:MasterIP}:443 --cluster-dns=`$global:KubeDnsServiceIp --cluster-domain=cluster.local  --kubeconfig=c:\k\config --hairpin-mode=promiscuous-bridge --v=2 --azure-container-registry-config=c:\k\azure.json
"@
```
And in `Write-KubernetesStartFiles`
```
function
Write-KubernetesStartFiles($podCIDR)
{
    if ($global:KubeBinariesVersion -eq "1.5.3")
    {
        $KubeletArgList = $global:KubeletArgListBase
        $KubeletCommandLine = $global:KubeletCommandLineBase
    }
    else
    {
        $KubeletArgList = $global:KubeletArgListBase += "--enable-cri=false"
        $KubeletCommandLine = $global:KubeletCommandLineBase + "--enable-cri=false --image-pull-progress-deadline=20m"
    }
    $KubeletArgListStr = '@("{0}")' -f ($KubeletArgList -join '","')
```
Later
```
  `$argList = $KubeletArgListStr

  `$env:VIP_CIDR="10.0.0.0/8"
  $KubeletCommandLine
```